File: composicion-agregacion/Ejercicio2_java/empleado.py
class Empleado:
    def __init__(self, nombre, cargo, sueldo):
        self.__nombre = nombre
        self.__cargo = cargo
        self.__sueldo = sueldo
    def getSalario(self):
        return self.__sueldo
    
    def setSalario(self, nv):
        self.__sueldo = nv
    
    def getCargo(self):
        return self.__cargo
    
    def getNombre(self):
        return self.__nombre
    
    def __str__(self):
        return f"nombre: {self.getNombre()} cargo: {self.getCargo()} sueldo: {self.getSalario()}"

File: composicion-agregacion/Ejercicio2_java/java/Empleado.java
public class Empleado {
    private String nombre;
    private String cargo;
    private double sueldo;

    public Empleado(String nombre, String cargo, double sueldo) {
        this.nombre = nombre;
        this.cargo = cargo;
        this.sueldo = sueldo;
    }

    public double getSalario() {
        return this.sueldo;
    }

    public void setSalario(double nuevoSueldo) {
        this.sueldo = nuevoSueldo;
    }

    public String getCargo() {
        return this.cargo;
    }

    public String getNombre() {
        return this.nombre;
    }

    @Override
    public String toString() {
        return "nombre: " + this.getNombre() + " cargo: " + this.getCargo() + " sueldo: " + this.getSalario();
    }
}
